model/commitqueue/commit_queue.go

```diff
@@ -9,10 +9,8 @@ const (
 )
 
 type CommitQueue struct {
-	ProjectID    string   `bson:"_id"`
-	Queue        []string `bson:"queue"`
-	MergeAction  string   `bson:"merge"`
```

Comment:
Moved MergeAction and StatusAction into the commit queue parameters in projectRef